libraries/stdlib/samples/test/samples/system/timing/timing.kt

```diff
@@ -0,0 +1,26 @@
+package samples.system.timing
+
+import samples.*
+import kotlin.system.*
+import kotlin.test.*
+
+class Timings {
+
+    @Sample
+    fun measureBlockTimeMillis() {
+        val timeInMillis = measureTimeMillis {
+            (0..10).reduce(Int::plus)
+        }
+        assertTrue(timeInMillis >= 0)
+        assertTrue(timeInMillis < 1e9)
```

Comment:
I think such assertions are not that informative for the purpose of sample. Would be better just to print out the value, something like `println("The operation took $timeInMillis ms")`

Reply:
Good suggestions. I'll try that.